pylsp/plugins/rope_autoimport.py

```diff
@@ -0,0 +1,240 @@
+import logging
+from collections import OrderedDict
+from functools import lru_cache
+from typing import Generator, List, Set
+
+import parso
+from parso.python import tree
+from parso.tree import NodeOrLeaf
+from rope.base.resources import Resource
+from rope.contrib.autoimport.sqlite import AutoImport
+
+from pylsp import hookimpl
+from pylsp.config.config import Config
+from pylsp.workspace import Document, Workspace
+
+log = logging.getLogger(__name__)
+
+
+_score_pow = 5
+_score_max = 10**_score_pow
+
+
+@hookimpl
+def pylsp_settings():
+    # Default rope_completion to disabled
+    return {"plugins": {"rope_autoimport": {"enabled": True}}}
```

Comment:
Should be disabled by default probably? Both here and in the schema and documentation.

Reply:
Done